tests/conftest.py

```diff
@@ -63,6 +75,9 @@
         "https://raw.githubusercontent.com/bioimage-io/spec-bioimage-io/main/example_specs/models/"
         "stardist_example_model/rdf_wrong_shape2.yaml"
     ),
+    "shape_change": (
+        "/home/skar/Work/Code/Packages/core-bioimage-io-python/shape_change_model.yaml"
```

Comment:
Can you move the model to `example_specs/models` and replace the local filepath with the url?

Reply:
I don't see an `example_specs/models` directory, is that in another repo?

Reply:
the examples are in the spec folder where you'd have to make a separate PR..
https://github.com/bioimage-io/spec-bioimage-io/tree/main/example_specs/models

Reply:
Thanks for clarifying @FynnBe. Indeed, the examples would need to go into the other repository.

Reply:
:thumbsup: Thanks!